courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2007 @@
+{
```

Comment:
<p>I get an error here when importing</p><p>"from __future__ import absolute_import, \</p><p>&nbsp;&nbsp;division, print_function, unicode_literals"</p><p>Can be fixed by moving this import to the top of the cell.</p><p> </p><p>Do we need these imports? I think they are new from last time?</p><p> </p><p>Also, I don't think we use EarlyStopping below, or ModelCheckpoint, or TensorBoard. Did you want to? If not, I'd remove them from imports.</p><p>Same with importing 'plot_model' and 'MinMaxScaler'. Looks like you do MinMax Scaling manually, which is fine. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Removed the unused statements.  When I first began working on the Notebook, I had thought to add EarlyStopping and TensorBoard - but that would have distracted from the core goal of the lab - e.g. feature engineering.</p>